pdns/recursordist/docs/manpages/rec_control.1.rst

```diff
@@ -92,6 +92,9 @@ dump-cache *FILENAME*
     also dumped to the same file. The per-thread positive and negative cache
     dumps are separated with an appropriate comment.
 
+dump-dot-probe-map *FILENAME*
+    Dump the contents of the DoT probe map to the *FILENAME* mentioned.
+
 dump-edns *FILENAME*
     Dumps the EDNS status to the filename mentioned. This file should not exist
```

Comment:
Not your code, but it probably should use `*FILENAME*` as you did.

I'm not a fan of _mentioned_, I'd use `specified FILENAME `, but it appears to be house style.